docs/UserGuide.adoc

```diff
@@ -542,52 +542,61 @@ then annotations are shown as well.
 
 === Adding reminder: `reminder`
 
-Adds a reminder to visit a bookmark. You can set the time for the reminder. The app
-will inform you at that time. You can also choose whether to open the bookmark.
+Adds a reminder to visit a bookmark. You can set the time for the reminder.
+You can also choose whether to open the bookmark.
 The default setting is open.
 
-Format: `reminder INDEX t/TIME_DATE [n/NOTE=Open]`
+Format: `reminder INDEX t/TIME_DATE [d/DESCRIPTION] [n/NOTE=Open]`
 
 * INDEX is the index of the bookmark that you want to add with a reminder.
  * TIME_DATE follows HHMM dd/mm/yyyy format.
+* DESCRIPTION is the description of the reminder.
 * NOTE is the setting of whether you want to open the bookmark.
 
 Examples:
 
 * `reminder 1 t/2359 02/07/2019` +
-Adds a reminder for bookmark 1, and set the time at 02/07/2019. The
-reminder will remind you at 23:59 on 2nd July, 2019.
+Adds a reminder for bookmark 1, and set the time at 02/07/2019.
 It will also open the bookmark for you.
 
 * `reminder 1 t/2359 02/07/2019 n/Close` +
-Adds a reminder for bookmark 1, and set the time at 23:59 of 02/07/2019. The reminder will
-remind you at the time and would not open the bookmark for you.
+Adds a reminder for bookmark 1, and set the time at 23:59 of 02/07/2019.
+The reminder would not open the bookmark for you.
```

Comment:
Another suggestion, to improve clarity. May not be needed if the parameter is changed.
```suggestion
The reminder will not open the bookmark at the specified time.
```

Reply:
Changed the parameter.